lib/telegraf/sidekiq.rb

```diff
@@ -56,7 +56,10 @@ def call(worker, job, queue)
         }
 
         values = {
-          retry_count: job['retry_count']
+          retry_count: job['retry_count'],
+          total_workers_count: ::Sidekiq.options[:concurrency],
+          workers_running_a_job: ::Sidekiq::Workers.new.size,
+          latency: ::Sidekiq::Queue.new.latency.to_f
```

Comment:
This only measures latency of the `default` queue. We use different queues in different services / workers.

Reply:
What _exactly_ is `::Sidekiq::Queue.new.latency` measuring? What latency? Redis command latency? The time between job queuing and processing? With or without retries? What about named queues, or multiple queues?

Do we have a definition here, or do we want to implement it ourselves to know what exactly is measured?

Reply:
The metric point is about a specific job. I would definitely expect the latency to be strictly related to the job only too.